flink-clients/src/main/java/org/apache/flink/client/program/ClusterClient.java

```diff
@@ -197,26 +197,33 @@ public LeaderConnectionInfo getClusterConnectionInfo() throws LeaderRetrievalExc
 	// ------------------------------------------------------------------------
 
 	public static String getOptimizedPlanAsJson(Optimizer compiler, PackagedProgram prog, int parallelism)
-			throws CompilerException, ProgramInvocationException {
+		throws CompilerException, ProgramInvocationException {
 		PlanJSONDumpGenerator jsonGen = new PlanJSONDumpGenerator();
 		return jsonGen.getOptimizerPlanAsJSON((OptimizedPlan) getOptimizedPlan(compiler, prog, parallelism));
 	}
 
 	public static FlinkPlan getOptimizedPlan(Optimizer compiler, PackagedProgram prog, int parallelism)
-			throws CompilerException, ProgramInvocationException {
-		Thread.currentThread().setContextClassLoader(prog.getUserCodeClassLoader());
-		if (prog.isUsingProgramEntryPoint()) {
-			return getOptimizedPlan(compiler, prog.getPlanWithJars(), parallelism);
-		} else if (prog.isUsingInteractiveMode()) {
-			// temporary hack to support the optimizer plan preview
-			OptimizerPlanEnvironment env = new OptimizerPlanEnvironment(compiler);
-			if (parallelism > 0) {
-				env.setParallelism(parallelism);
-			}
+		throws CompilerException, ProgramInvocationException {
```

Comment:
Same here with the formatting.

Reply:
Is there any style check plugin per Flink codebase I can use to avoid this? My editor moves throws to new line without double indentation

Reply:
No, unfortunately not. Ideally we would have an auto formatter configuration which we could simply share in the community. However, the community never managed to agree on a consistent code style.